env.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/bash
```

Comment:
```
# -*- coding: utf-8 -*-
#
```

Reply:
Also sh scripts require `-*- coding: utf-8 -*-`? I thought it was only necessary for python ones.

Reply:
Let's leave it just for Python for now -- but ultimately it should be for any scripting file.

Reply:
I checked other repos with licensing that has been added, and shell scripts do not have the utf-8 coding. I assumed to do the same in here. Closing comment